create-doi-request/build.gradle

```diff
@@ -0,0 +1,41 @@
+repositories {
+    maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
+}
+
+dependencies {
+
+    compileOnly project(":storage-model")
```

Comment:
Choose compile/test following each other, or block of compile/test. Annotate to explain why, because it is still confusing.

Reply:
fixed